node/node.d.ts

```diff
@@ -170,6 +170,7 @@ declare module NodeJS {
         removeAllListeners(event?: string): EventEmitter;
         setMaxListeners(n: number): void;
         listeners(event: string): Function[];
+        _events?:string[];
```

Comment:
Agreed that it does indeed exist ... however we do not support *public* access to internal implementations so this cannot be merged. 

Docs : https://nodejs.org/api/events.html